includes/class-wc-google-analytics-info-banner.php

```diff
@@ -1,114 +0,0 @@
-<?php
```

Comment:
Curious why this file is getting removed. I can see in trunk that it's no longer referenced, but I don't see a specific reasoning why it was removed.

Looks like it was removed in this PR #235. Would be nice to mention it in the PR description since it's not related to the changes.